### PR TITLE
fix(DB/Instance): Fix equip issue and text errors

### PR DESCRIPTION
**Pull request by Voxstrasza**
## CHANGES PROPOSED:
-  Remove Frostmourne as a base equipped item for The Lich King (ID: 37226), add SmartScript to equip Frostmourne when spell Frostmourne Equip (ID: 72729) is used by the cpp
- Unsheathe Frostmourne when The Lich King (ID: 36954) is fighting Jaina/Sylvanas
- Fix spelling errors and incorrect text so it matches the voice lines before the Escape from Arthas gauntlet event

## ISSUES ADDRESSED:
- When initiating the roleplay for Halls of Reflection, The Lich King walks out with Frostmourne already in-hand, when he should walk in unarmed, and walk out with the sword.
- The Lich King fights Jaina/Sylvanas unarmed
- Spelling errors and text errors from The Lich King, Jaina and Sylvanas.
- [Frostmourne before fix](https://youtu.be/87imZWKtO70)

## TESTS PERFORMED:
- Applied SQL
- Entered Halls of Reflection and initiated roleplay for Falric & Marwyn
- Completed Falric & Marwyn, went to Shadow Throne to watch fixed roleplay text and Frostmourne in The Lich King's hand
- [Frostmourne after fix](https://youtu.be/QjXVfqd10Kw)

## HOW TO TEST THE CHANGES:
Unfortunately, because of the attunements, you will have to attune yourself via completing the Forge of Souls & Pit of Saron first.  

**DO NOT** enter with `.gm on` - you will load the incorrect NPCs and cannot complete the quests or see the fixes

Apply SQL, restart server, create two characters - one horde, one alliance, and `.level 79` to become level 80.

**Alliance:**

1. `.quest add 24510`
2. `.tele FrozenHalls`
3. Enter Forge of Souls, `.mod speed 10`
4. `.learn 27261`, put it on easy hotkey - kill everything, complete the quests
5. Enter Pit of Saron, repeat step 4
6. Enter Halls of Reflection, complete roleplay, watch how Arthas enters the room unarmed and equips Frostmourne when he reaches for it
7. Complete Falric, Marwyn and all trash mobs
8. Go to the Shadow Throne, confirm that Arthas has Frostmourne equipped, and read the updated text - [compare with this](https://wow.gamepedia.com/Escape_from_Arthas#Quotes)

**Horde:**

1. `.quest add 24506`
2. `.tele FrozenHalls`
3. Enter Forge of Souls, `.mod speed 10`
4. `.learn 27261`, put it on easy hotkey - kill everything, complete the quests
5. Enter Pit of Saron, repeat step 4
6. Enter Halls of Reflection, complete roleplay, watch how Arthas enters the room unarmed and equips Frostmourne when he reaches for it (Jaina and Uther have scuffed up scripts here, I'm working on fixing that for another PR)
7. Complete Falric, Marwyn and all trash mobs
8. Go to the Shadow Throne, confirm that Arthas has Frostmourne equipped, and read the updated text - [compare with this](https://wow.gamepedia.com/Escape_from_Arthas#Quotes)

Finally, compare the way Frostmourne is handled [with this video.](https://youtu.be/izx-lkkSKA4?t=187)

## Target branch(es):
- [x] Master

### CODE REVIEW

**Comment by Voxstrasza** [comment]
@FrancescoBorzi why does keira3 add so many ()?

**Comment by FrancescoBorzi** [comment]
@Voxstrasza because of the library [squel](https://hiddentao.github.io/squel/) we use to generate some of the SQL queries, not sure if there is a way to avoid that. It's not an issue either.

**Comment by Voxstrasza** [comment]
> 
> 
> @Voxstrasza because of the library [squel](https://hiddentao.github.io/squel/) we use to generate some of the SQL queries, not sure if there is a way to avoid that. It's not an issue either.

ah okay, was just curious

File: data/sql/updates/pending_db_world/rev_1606373029735379400.sql [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+UPDATE `creature_text` SET `Text`='He\'s... too powerful. Heroes, quickly... come to me! We must leave this place at once! I will do what I can do hold him in place while we flee.' WHERE  `CreatureID`=37554 AND `GroupID`=0 AND `ID`=0;
+UPDATE `creature_text` SET `Text`='He\'s... too powerful. Heroes, quickly... come to me! We must leave this place at once! I will do what I can do hold him in place while we flee.' WHERE  `CreatureID`=37554 AND `GroupID`=102 AND `ID`=0;
+UPDATE `broadcast_text` SET `FemaleText`='He\'s... too powerful. Heroes, quickly... come to me! We must leave this place at once! I will do what I can do hold him in place while we flee.' WHERE  `ID`=37168;
+UPDATE `broadcast_text` SET `FemaleText`='He is too powerful, we must leave this place at once! My magic will hold him in place for only a short time! Come quickly, heroes!' WHERE  `ID`=36821;
+UPDATE `broadcast_text` SET `MaleText`='Your allies have arrived Jaina, just as you promised. You will all become powerful agents of the Scourge.' WHERE  `ID`=37172;
```
Comment (Si1ker) [inline data/sql/updates/pending_db_world/rev_1606373029735379400.sql:22]: new line at the end
```suggestion
UPDATE `broadcast_text` SET `MaleText`='Your allies have arrived Jaina, just as you promised. You will all become powerful agents of the Scourge.' WHERE  `ID`=37172;

```

**Review by Si1ker — APPROVED** [review]
The videos attached to the PR description are enough testing for me.

/* Affected rows: 13  Found rows: 0  Warnings: 0  Duration for 14 queries: 0.031 sec. */

**Pull request merged**